rasa/nlu/test.py

```diff
@@ -411,31 +427,34 @@ def evaluate_entities(
 
     aligned_predictions = align_all_entity_predictions(entity_results, extractors)
     merged_targets = merge_labels(aligned_predictions)
-    merged_targets = substitute_labels(merged_targets, "O", "no_entity")
+    merged_targets = substitute_labels(merged_targets, "O", NO_ENTITY)
 
     result = {}
 
     for extractor in extractors:
         merged_predictions = merge_labels(aligned_predictions, extractor)
-        merged_predictions = substitute_labels(merged_predictions, "O", "no_entity")
+        merged_predictions = substitute_labels(merged_predictions, "O", NO_ENTITY)
         logger.info("Evaluation for entity extractor: {} ".format(extractor))
         if report_folder:
-            report, precision, f1, accuracy = get_evaluation_metrics(
-                merged_targets, merged_predictions, output_dict=True
-            )
-
             report_filename = extractor + "_report.json"
             extractor_report_filename = os.path.join(report_folder, report_filename)
 
+            report, precision, f1, accuracy = get_evaluation_metrics(
+                merged_targets,
+                merged_predictions,
+                output_dict=True,
+                include_no_entity=False,
+            )
             utils.write_json_to_file(extractor_report_filename, report)
+
             logger.info(
                 "Classification report for '{}' saved to '{}'."
                 "".format(extractor, extractor_report_filename)
             )
 
         else:
             report, precision, f1, accuracy = get_evaluation_metrics(
-                merged_targets, merged_predictions
+                merged_targets, merged_predictions, include_no_entity=False
```

Comment:
should we explicitly say `output_dict=False,`? took me some time to get the difference

Reply:
No need for that, default is `False`.

Reply:
yes, for clarification